.github/workflows/docs.yml

```diff
@@ -20,6 +20,7 @@ jobs:
         pip install -e ".[shell]"
         pip install -r requirements/requirements-test.txt
         pip install -r requirements/requirements-docs.txt
+        pip install -r requirements/requirements-extras-sdk.txt
```

Comment:
I think `sdk` is a bit unclear. Can we do `sagemaker-sdk`?

Reply:
Good idea